behavior_tree_core/package.xml

```diff
@@ -31,6 +31,8 @@
   <run_depend>roscpp</run_depend>
```

Comment:
Update to using [package format="2"](http://docs.ros.org/jade/api/catkin/html/howto/format2/index.html).